tests/textlab_api/textlab_api_test.py

```diff
@@ -0,0 +1,345 @@
+from __future__ import annotations
+
+import json
+import time
+from typing import TYPE_CHECKING
+
+import pytest
+from django.http import HttpResponse
+from django.test.client import Client
+from django.urls import reverse
+from werkzeug.wrappers import Request, Response
+
+from integreat_cms.cms.constants import status
+from integreat_cms.cms.constants.roles import EDITOR
+from integreat_cms.cms.models.pages.page import Page
+from integreat_cms.cms.models.pages.page_translation import PageTranslation
+from integreat_cms.cms.models.regions.region import Region
+
+if TYPE_CHECKING:
+    from django.test.client import Client
+    from pytest_django.fixtures import SettingsWrapper
+    from pytest_httpserver.httpserver import HTTPServer
+
+
+def update_page_content(
+    login_role_user: tuple[Client, str],
+    title: str,
+    content: str,
+    hix_ignore: bool = False,
+) -> tuple[str, HttpResponse]:
+
+    # Log the user in
+    client, _role = login_role_user
+
+    # Update page content
+    edit_page = reverse(
+        "edit_page",
+        kwargs={
+            "region_slug": "augsburg",
+            "language_slug": "de",
+            "page_id": 2,
+        },
+    )
+    edit_page_data = {
+        "title": title,
+        "content": content,
+        "mirrored_page_region": "",
+        "_ref_node_id": 3,
+        "_position": "right",
+        "status": status.PUBLIC,
+    }
+    if hix_ignore:
+        edit_page_data["hix_ignore"] = "on"
+
+    return (edit_page, client.post(edit_page, edit_page_data))
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("login_role_user", [EDITOR], indirect=True)
+def test_hix_score_update(
+    load_test_data: None,
+    login_role_user: tuple[Client, str],
+    settings: SettingsWrapper,
+    httpserver: HTTPServer,
+) -> None:
```

Comment:
```suggestion
def test_hix_score_update(
    load_test_data: None,
    admin_client: Client,
    settings: SettingsWrapper,
    httpserver: HTTPServer,
) -> None:
```
I suggest using admin_client in these tests instead of login_role_user because we're testing hix_signals that trigger regardless of user role anyway.
If we use admin_client here, it will just simplify the code a little bit.

The update_page_content() function and all other tests should be updated accordingly then.